### PR TITLE
Rodriguez_Roberto

### PR DESCRIPTION
**Pull request by robertorodriguez12**

### CODE REVIEW

**Comment by KathleenScriver** [comment]
Would love to see more commits. After every test there should be a commit, and then after each method/code written that gets that test green, another commit.

File: lib/author.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Author
+
+  attr_reader :first_name,
```
Comment (KathleenScriver) [inline lib/author.rb:3]: If you add extra methods, be sure to add tests for those methods too.

**Comment by robertorodriguez12** [comment]
Yeah, I forgot to make the commits after I finished tests and methods. I will remember to do the commits next time.

File: lib/author.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  def write(title, publication_date)
+    book_info = {author_first_name: @first_name,
+                author_last_name: @last_name,
+                title: title,
+                publication_date: publication_date}
+    # binding.pry
+  book = Book.new(book_info)
+  add_book(book)
+  return book
```
Comment (KathleenScriver) [inline lib/author.rb:26]: Great job!!! This is perfect!

File: lib/book.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  def author
+    @author_first_name + " " + @author_last_name
+  end
+
+  def publication_year
+    # binding.pry
+    publication_date_arr = @publication_date.split(",")
+    publication_year_info = publication_date_arr[1].strip
```
Comment (KathleenScriver) [inline lib/book.rb:23]: Works for now until we don't have a comma later on in the interaction pattern.

File: test/author_test.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                                  last_name: "Bronte"})
+    assert_equal [], charlotte_bronte.books
+  end
+
+  def test_author_can_write_book
+    charlotte_bronte = Author.new({first_name: "Charlotte",
+                                  last_name: "Bronte"})
+
+    jane_eyre = charlotte_bronte.write("Jane Eyre", "October 16, 1847")
+    assert_equal Book, jane_eyre.class
```
Comment (KathleenScriver) [inline test/author_test.rb:32]: Awesome, great job testing the return value for this method, as well as all the side effects.

**Pull request closed without merging**